src/main/java/io/orten/nano/business/OrganizationAPI.java

```diff
@@ -57,9 +57,9 @@ public Response getAllOrg(){
      * deletes an organization object from the database
      */
     @DELETE
-    @Path("{orgID}")
-    public Response deleteOrg(@PathParam("orgID") String orgID){
-        return new OrganizationService().delete(orgID);
+    @Path("{orgId}")
+    public Response deleteOrg(@PathParam("orgID") Long orgId){
```

Comment:
Does this really work?
either way the path parameter should match the path, even if it is case-insensitive